pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -195,6 +195,34 @@ type KeycloakAPIRealm struct {
 	// User Managed Access Allowed
 	// +optional
 	UserManagedAccessAllowed *bool `json:"userManagedAccessAllowed,omitempty"`
+
+    // OTP Policy Algorithm
+    // +optional
+	OtpPolicyAlgorithm string `json:"otpPolicyAlgorithm,omitempty"`
+
+	// OTP Policy Digits
```

Comment:
Can you please fix the indentation here?

Reply:
Should be fixed.